backend/controller/admin/cmd_client.go

```diff
@@ -0,0 +1,76 @@
+package admin
+
+import (
+	"context"
+	"errors"
+	"net"
+	"net/url"
+
+	"connectrpc.com/connect"
+	ftlv1 "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1"
+	"github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1/ftlv1connect"
+)
+
+type CmdClient interface {
```

Comment:
Add a comment here describing what this is for

Reply:
I think just call this `Client`

Reply:
Ahh thanks for that reminder. Comment added and rename done!